packages/dx-core/src/utils.ts

```diff
@@ -23,6 +21,12 @@ export const insertPlugin = (array, newItem) => {
   return result;
 };
 
+/** @internal */
+export const removePlugin = (array, item) => {
+  const i = array.indexOf(item);
+  return i >= 0 ? [...array.slice(0, i), ...array.slice(i + 1)] : array;
```

Comment:
Can we change the `i` name to `itemIndex` or something more clearly?

Reply:
Yes, we can.